docs/sources/developer/architecture/components/internal-logger.md

```diff
@@ -0,0 +1,18 @@
+# Internal Logger
+
+The internal logger is a helper utility that is used to log messages from the library itself. Due to the fact that it
+relies on the unpatched console, the messages printed by the internal logger are not captured during
+auto-instrumentation of the `console` object.
```

Comment:
Add an example of what that looks like vs the normal console logging

```javascript
// This will get logged and ingested 
console.log('Example');

// This will not
internalLogger.info('Example');
```